debian/control

```diff
@@ -13,9 +13,9 @@ Depends: ${shlibs:Depends}, ${misc:Depends}, python3, python3-pyqt5, python3-pyq
 Description: securedrop client for qubes workstation
 
 Package: securedrop-export
-Architecture: all
-Depends: ${misc:Depends}, python3, udisks2, cups, printer-driver-brlaser, printer-driver-hpcups, system-config-printer, xpp, libcups2, gnome-disk-utility, libreoffice,
```

Comment:
Just FYI xpp will still be present in the template, it doesn't get automatically removed. I think we might need to do it via salt? Not sure if a conflict will work with the Qubes updater. But we can figure that out after this release.